kato/kato-cf/src/main/groovy/com/netflix/spinnaker/kato/cf/deploy/validators/AbstractEnableDisableCloudFoundryServerGroupDescriptionValidator.groovy

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2015 Google, Inc.
```

Comment:
s/Google/Cloud Foundry
(Also noticed that a lot of the files don't have copyright headers.)